components/Editor.js

```diff
@@ -66,12 +63,11 @@ class Editor extends React.Component {
 
   componentDidMount() {
     // Load from localStorage instead of query params
-    if (!this.props.initialState) {
-      const state = getState(localStorage)
-      if (state) {
-        this.setState(state)
-      }
-    }
+    this.setState({
+      ...this.state,
```

Comment:
You don't need to include `this.state` in `setState` (:

Reply:
Fixed